### PR TITLE
Review  submission

### PR DESCRIPTION
**Pull request by Visvanath27**
please review

### CODE REVIEW

File: templates/Reviews.html [hunk 1]
```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
```
Comment (rishikaharithaMSIT) [inline templates/Reviews.html:1]: Will reviews appear in separate page or in book page?

File: templates/Reviews.html [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+<html lang="en">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+<!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"> -->
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```
Comment (rishikaharithaMSIT) [inline templates/Reviews.html:9]: We can make use of layout.html, thats already existing

File: templates/Reviews.html [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+            </div>
+        </div>
+    </div>
+    {% endfor %}
+    </div>
+
+    <style>
+        :root {
+            --image_size: 50px;
+        }
```
Comment (rishikaharithaMSIT) [inline templates/Reviews.html:152]: Move css to separate folder

File: application.py [hunk 4]
```diff
@@ -3,12 +3,16 @@
 
 from flask import Flask, session,render_template, request,flash, redirect, url_for
 from flask_session import Session
+from flask_humanize import Humanize
```
Comment (rishikaharithaMSIT) [inline application.py:6]: what is this humanize for?

File: templates/Reviews.html (hunk 1 above)
Comment (Visvanath27) [reply templates/Reviews.html:1]: erviews appear in book page while merging will make the changes needed

File: application.py (hunk 4 above)
Comment (Visvanath27) [reply application.py:6]: humanize is for day display of the review.

File: templates/Reviews.html (hunk 3 above)
Comment (Visvanath27) [reply templates/Reviews.html]: changed and created a new css file.

File: templates/Reviews.html (hunk 1 above)
Comment (rishikaharithaMSIT) [reply templates/Reviews.html]: you can change now, by the end of this review code should be ready to merge

File: application.py (hunk 4 above)
Comment (rishikaharithaMSIT) [reply application.py:6]: can you elaborate ?

File: templates/Reviews.html (hunk 1 above)
Comment (Visvanath27) [reply templates/Reviews.html]: I have changed the name of the file with respect to the book page author.

File: application.py (hunk 4 above)
Comment (Visvanath27) [reply application.py:6]: Humanize is a flask module which changes the date and time format in which we wish to display.
so I have used this module in displaying reviews mentioning when the review is posted by user like Today, Tomorrow, Yesterday.

**Pull request merged**